.github/workflows/test_gpu_cu11.yml

```diff
@@ -0,0 +1,325 @@
+name: Testing::Linux::x86_64 (Lightning-GPU)
```

Comment:
Rename `tests_gpu_cu11.yml` to have `tests_` prefix for all workflows.

Reply:
Done!